testdata/zk/InvariantDeposit.t.sol

```diff
@@ -5,25 +5,56 @@ import "ds-test/test.sol";
 import "../cheats/Vm.sol";
 import "./Deposit.sol";
 
-contract ZkInvariantTest is DSTest {
+// partial from forge-std/StdInvariant.sol
+abstract contract StdInvariant {
+    struct FuzzSelector {
+        address addr;
+        bytes4[] selectors;
+    }
+
+    address[] internal _targetedContracts;
+
+    function targetContracts() public view returns (address[] memory) {
+        return _targetedContracts;
+    }
+
+    FuzzSelector[] internal _targetedSelectors;
+
+    function targetSelectors() public view returns (FuzzSelector[] memory) {
+        return _targetedSelectors;
+    }
+
+    address[] internal _targetedSenders;
+
+    function targetSenders() public view returns (address[] memory) {
+        return _targetedSenders;
+    }
+}
+
```

Comment:
yeah I could define those there but I split it to keep it clearer why those exist, also we still define all the items in the setUp, so it's close to the test anyways

Reply:
It's fine in general, just looks weird with the `push` because we can simply return the static values directly from the functions, without storage or mutation.